config/config.php

```diff
@@ -524,35 +524,35 @@
 
 		'22' => [
 			'100' => [
-				'latest' => '22.2.0',
-				'internalVersion' => '22.2.0.2',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-22.2.0.zip',
+				'latest' => '22.2.1 RC1',
+				'internalVersion' => '22.2.1.0',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-22.2.1rc1.zip',
 				'web' => 'https://docs.nextcloud.com/server/22/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
 				'minPHPVersion' => '7.3',
-				'signature' => 'IF7GWr7KVXkakHrQixMoXXQziCFMmUNI42fuCbgQLsv1+s8f6Ezaux7VlsO8FRAm
-dPkyZq2MNLdZYeWtTTI38xoiqpsaJA8eR8HXEA+KJ83egH0C5K1SacPFsRyJipyc
-cynghWwswoHsLht1ms2E6LWPelc4SaIL6Ngitv+1Vxjw7+T6mHzV4/5y7Rc4AFPG
-XFdlY91U6sxuhgN9Uti7JJWNeu1V6wSjnzGUqR5egkSeyL+B76TNezhGfbqvVZbb
-AUTy0ZHZXAt9LIGI9YmqBv90CgmW5xmBrt/qqw0f/M4A4C1BCDwh4DI2Ylh5OS+R
-igCw9M1FBIhT4+LRdFBupQ==',
+				'signature' => 'gFblqxh+w7sGgWkYa/joydKyexmP7g2lZmljjXdifxPk9vf167UGAK2nQgHgge15
+rqBiBPDne82q7ysPBSBBzzw2qyWYK4Y629zcDVEurOxsxV/yzh2lrvPZiCrrHltG
+0cMqyOeUVBtKGGIEp34LYtG7M1wr0b92Jy1IMzanFP8NAG8/N4pYNevRm1JjB9BY
+NVp19RztO8m/mPuRFTXSNZ1SDC6C3yMWrST07rjHOymZJSa8CJyroK/BfQdVonS/
+EN0BxBBUpY5/ZBOvCIBIy0LcP/JpgTkdCim2ooxbAwAof1yvTuubkTGmHQNzoVnv
+ITFtk0PeXc1HZzFZ65CdQw==',
 			],
 		],
 
 		'21.0.5' => [
```

Comment:
Will change once we have 21.0.6 stable?

Reply:
beta should feed beta

Reply:
What does that means? Should I change it now?